jersey/src/main/java/org/pac4j/jax/rs/jersey/features/Pac4JValueFactoryProvider.java

```diff
@@ -196,19 +196,18 @@ protected void configure() {
 
             if(manager == null){
                 bind(DefaultProfileManagerFactoryBuilder.class)
-                    .to(ProfileManagerFactoryBuilder.class)
-                ;
+                    .to(ProfileManagerFactoryBuilder.class).in(Singleton.class);
             } else {
                 bind(manager).to(ProfileManagerFactoryBuilder.class);
             }
 
-            bind(Pac4JProfileValueFactoryProvider.class).to(ValueParamProvider.class).in(Singleton.class);
+            bindAsContract(Pac4JProfileValueFactoryProvider.class).to(ValueParamProvider.class).in(Singleton.class);
```

Comment:
these need to be bindAsContract so we bind them to their own class as well as the interfaces we have here, this is what was causing the errors

Reply:
Thx. Does it sill work for Dropwizard v1?

Reply:
v1 would use the jersey225 module of this repo so yes v1 should not be effected by this change

Reply:
OK. Good. Let's merge. I guess we need a new `jax-rs-pac4j` release then?